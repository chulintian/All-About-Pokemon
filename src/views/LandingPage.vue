<template>
    <div>

        <div class="p-3">
            <div class="container-fluid">
                <div class="row">
                    <div class="col-6 col-md-3 p-2">
                        <div class="bg-gold-yellow rounded-3 d-flex justify-content-center align-items-center p-3 h-100">
                            <img src="../assets/pokeball.png" alt="pokeball pic" class="w-75"/>
                        </div>
                    </div>
                    <div class="col-6 col-md-3 p-2">
                        <div class="bg-blue rounded-3 d-flex justify-content-center align-items-center p-3 h-100">
                            <img src="../assets/charmander.png" alt="charmander pic" class="w-75"/>
                        </div>
                    </div>
                    <div class="col-6 col-md-3 p-2">
                        <div class="bg-gold-foil rounded-3 d-flex justify-content-center align-items-center p-3 h-100">
                            <img src="../assets/snorlax.png" alt="snorlax pic" class="w-75"/>
                        </div>
                    </div>
                    <div class="col-6 col-md-3 p-2">
                        <div class="bg-red rounded-3 d-flex justify-content-center align-items-center p-3 h-100">
                            <img src="../assets/pikachu.png" alt="pikachu pic" class="w-75"/>
                        </div>
                    </div>
                </div>
            </div>
    
            <div class="d-flex justify-content-center">
                <h2 class="text-center p-3">Join us on a Pokemon Adventure NOW</h2>
            </div>
    
            <div class="container-fluid">
                <div class="row">
                    <div class="col-6 col-sm-4 col-md-3 p-2">
                        <div class="bg-red rounded-3 d-flex justify-content-center align-items-center p-3 h-100">
                            <img src="../assets/evee.png" alt="evee pic" class="w-75"/>
                        </div>
                    </div>
                    <div class="col-6 col-sm-4 col-md-3 p-2">
                        <div class="bg-gold-foil rounded-3 d-flex justify-content-center align-items-center p-3 h-100">
                            <img src="../assets/corsola.png" alt="corsola pic" class="w-75"/>
                        </div>
                    </div>
                    <div class="col-6 col-sm-4 col-md-3 p-2">
                        <div class="bg-blue rounded-3 d-flex justify-content-center align-items-center p-3 h-100">
                            <img src="../assets/mainguy.png" alt="main guy pic" class="w-50"/>
                        </div>
                    </div>
                    <div class="col-6 col-sm-4 col-md-3 p-2">
                        <div class="bg-gold-yellow rounded-3 d-flex justify-content-center align-items-center p-3 h-100">
                            <img src="../assets/swadloon.png" alt="swadloon pic" class="w-75"/>
                        </div>
                    </div>
                </div>
            </div>
        </div>

        

    </div>
</template>
  
<script>

export default {
name: 'LandingPage',
    components: {
    },
}

</script>

<style>

.bg-red {
background-color: #CC0000;
}

.bg-gold-yellow {
background-color: #FFDE00;
}

.bg-gold-foil {
background-color: #B3A125;
}

.bg-blue {
background-color: #3B4CCA;
}


</style>
