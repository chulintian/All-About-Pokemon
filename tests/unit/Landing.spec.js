import { shallowMount } from "@vue/test-utils";
import LandingPage from "../../src/views/LandingPage.vue";

describe("LandingPage", () => {
  it("renders correctly", () => {
    const wrapper = shallowMount(LandingPage);
    expect(wrapper.element).toMatchSnapshot();
  });
});
