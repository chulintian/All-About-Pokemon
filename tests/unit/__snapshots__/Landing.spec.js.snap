// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage renders correctly 1`] = `
<div>
  <div
    class="p-3"
  >
    <div
      class="container-fluid"
    >
      <div
        class="row"
      >
        <div
          class="col-6 col-md-3 p-2"
        >
          <div
            class="bg-gold-yellow rounded-3 d-flex justify-content-center align-items-center p-3 h-100"
          >
            <img
              alt="pokeball pic"
              class="w-75"
              src=""
            />
          </div>
        </div>
        <div
          class="col-6 col-md-3 p-2"
        >
          <div
            class="bg-blue rounded-3 d-flex justify-content-center align-items-center p-3 h-100"
          >
            <img
              alt="charmander pic"
              class="w-75"
              src=""
            />
          </div>
        </div>
        <div
          class="col-6 col-md-3 p-2"
        >
          <div
            class="bg-gold-foil rounded-3 d-flex justify-content-center align-items-center p-3 h-100"
          >
            <img
              alt="snorlax pic"
              class="w-75"
              src=""
            />
          </div>
        </div>
        <div
          class="col-6 col-md-3 p-2"
        >
          <div
            class="bg-red rounded-3 d-flex justify-content-center align-items-center p-3 h-100"
          >
            <img
              alt="pikachu pic"
              class="w-75"
              src=""
            />
          </div>
        </div>
      </div>
    </div>
    <div
      class="d-flex justify-content-center"
    >
      <h2
        class="text-center p-3"
      >
        Join us on a Pokemon Adventure NOW
      </h2>
    </div>
    <div
      class="container-fluid"
    >
      <div
        class="row"
      >
        <div
          class="col-6 col-sm-4 col-md-3 p-2"
        >
          <div
            class="bg-red rounded-3 d-flex justify-content-center align-items-center p-3 h-100"
          >
            <img
              alt="evee pic"
              class="w-75"
              src=""
            />
          </div>
        </div>
        <div
          class="col-6 col-sm-4 col-md-3 p-2"
        >
          <div
            class="bg-gold-foil rounded-3 d-flex justify-content-center align-items-center p-3 h-100"
          >
            <img
              alt="corsola pic"
              class="w-75"
              src=""
            />
          </div>
        </div>
        <div
          class="col-6 col-sm-4 col-md-3 p-2"
        >
          <div
            class="bg-blue rounded-3 d-flex justify-content-center align-items-center p-3 h-100"
          >
            <img
              alt="main guy pic"
              class="w-50"
              src=""
            />
          </div>
        </div>
        <div
          class="col-6 col-sm-4 col-md-3 p-2"
        >
          <div
            class="bg-gold-yellow rounded-3 d-flex justify-content-center align-items-center p-3 h-100"
          >
            <img
              alt="swadloon pic"
              class="w-75"
              src=""
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
